tasks/redhat.task/README.md

```diff
@@ -0,0 +1,19 @@
+# Task notes for Redhat 6
```

Comment:
Yes; `redhat` is a shortcut for `redhat/6`. `centos` is also a shortcut for `centos/6`.